### PR TITLE
2021 07 01 website again

### PR DESCRIPTION
**Pull request by richard-thonig**
Update the website to new CSP.guru version

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -5,17 +5,20 @@ title: Paper and Analysis
...
 
 ## Peer-reviewed publications leading to new csp.guru versions
 
 Lilliestam, J., L. Ollier, M. Labordena, S. Pfenninger and R. Thonig (2021). The near- to mid-term outlook for concentrating solar power: mostly cloudy, chance of sun, in: [Energy Sources, Part B: Economics, Planning, and Policy 16 (1): 23-41.](https://doi.org/10.1080/15567249.2020.1773580)
 
 Lilliestam, J. and R. Pitz-Paal (2018). Concentrating solar power for less than USD 0.07 per kWh: finally the breakthrough?, in: [Renewable Energy Focus 26: 17-21](https://doi.org/10.1016/j.ref.2018.06.002)
 
-Lilliestam, J., Labordena, M., Patt, A., & Pfenninger, S. (2017). Empirically observed learning rates for concentrating solar power and their responses to regime change. [Nature Energy, 2, 17094.]( https://doi.org/10.1038/nenergy.2017.94)
+Lilliestam, J., M. Labordena, A. Patt, & S. Pfenninger, (2017). Empirically observed learning rates for concentrating solar power and their responses to regime change. [Nature Energy, 2, 17094.]( https://doi.org/10.1038/nenergy.2017.94)
```
Comment (timtroendle) [inline paper.md:16]: This change doesn't make sense, does it?

**Review by timtroendle — APPROVED** [review]

File: paper.md (hunk 1 above)
Comment (richard-thonig) [reply paper.md:16]: why not? all the other entries have the initial first than the last name, only this one was the other way around?

Comment (timtroendle) [reply paper.md]: Oh ok, you are right! But are all the others correct? :P

It looks odd to me, but it may well be correct. Not a big issue in any case.

Comment (richard-thonig) [reply paper.md]: Maybe. I just copied them formated out of endnote. In any case. Consistency is key. If they are wrong, they should all be wrong in the same way :)

**Pull request merged**